Fixed the pargraph spelling mistake in 34_Specialty_Beer.xml

danbrough opened the pull request.

danbrough committed:
Fixed the pargraph spelling mistake in 34_Specialty_Beer.xml

SmedbergM commented:
It looks like this is an artifact of a [typo](https://github.com/SmedbergM/BJCP_2015/blob/e0089a0434d3367fadb9ec70cfd0624b14e0b638/build/make_style_XML.py#L162) in `build/make_style_XML.py`. Can you make the fix there and see if the XML builds correctly?

danbrough committed:
Fixed the pargraph spelling mistake in build/make_style_XML.py
and rebuilt the xml using a shell script that I also added (build/build.sh).

danbrough commented:
I fixed the error in `build/make_style_XML.py`  and also added a shell script to run the python scripts. 
Its at `build/build.sh`.

I've run this script and added the changes and it looks ok (better in fact as a lot of empty spaces at the end of paragraphs have been removed).

`xml/23_European_Sour_Ale.xml` and `xml/27_Historical_Beer.xml` look a bit different.

Will investigate further.

danbrough committed:
Fixed some syntax errors in the markdown files and regenerated the xml counterparts.
